SepulchureSaga/04ShadowfallRise.cs

```diff
@@ -0,0 +1,140 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+using RBot;
+
+public class ShadowfallRise
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new CoreFarms();
+    public CoreAdvanced Adv = new CoreAdvanced();
+    public CoreStory Story = new CoreStory();
+    public CoreDailies Daily = new();
+
+    public void ScriptMain(ScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(6592))
+        {
+            
+            Core.Logger("You have already completed ShadowfallRise Storyline");
+            return;
+        }
+
+        Story.PreLoad();
+
+        Core.EquipClass(ClassType.Solo);
+
+        //Fluid Dynamics 6539
```

Comment:
you DO NOT need the quest ids after the quest names here, only place you need quest names are inside the fuctions where they are called for :)